service/models.py

```diff
@@ -233,16 +240,23 @@ def create(self):
         logger.info("Creating %s, %s", self.last_name, self.first_name)
         # id must be none to generate next primary key
         self.id = None  # pylint: disable=invalid-name
+        # hash PWDs
+        self.password = hash_password(self.password)
         db.session.add(self)
         db.session.commit()
 
-    def update(self):
+    def update(self, original_password=None):
         """
         Updates a Customer to the database
         """
         logger.info("Updating/Saving %s, %s", self.last_name, self.first_name)
         if not self.id:
             raise DataValidationError("Update called with empty ID field")
+
+        # if PWD changed, hash again
+        if original_password is not None and not original_password == self.password:
+            self.password = hash_password(self.password)
```

Comment:
Can you add a test for this line so that the coverage is maintained?

Reply:
Done